testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1465,17 +1465,14 @@
   get_target('proxy').inject(file_path, file_path)
 end
 
-# TODO: Refactor this step to use the new mgrpxy command
-#       For now, this step is not used to onboard our containerized proxy
 When(/^I add avahi hosts in containerized proxy configuration$/) do
   if get_target('server').full_hostname.include? 'tf.local'
     hosts_list = ''
     $host_by_node.each do |node, _host|
       hosts_list += "--add-host=#{node.full_hostname}:#{node.public_ip} "
     end
     hosts_list = escape_regex(hosts_list)
-    regex = "s/^#?EXTRA_POD_ARGS=.*$/EXTRA_POD_ARGS=#{hosts_list}/g;"
-    get_target('proxy').run("sed -i.bak -Ee '#{regex}' /etc/sysconfig/uyuni-proxy-systemd-services")
+    get_target('proxy').run("echo 'export UYUNI_PODMAN_ARGS=\"#{hosts_list}\"' >> ~/.bashrc && source ~/.bashrc", runs_in_container: false)
```

Comment:
Question: normally, what goes into that file (.bashrc) is set up via sumaform (in controller module).

Is there a specific reason to do otherwise?

Reply:
This is proxy .bashrc, we don't have such mechanism for the proxy, only for the controller.
And personally, I find easier to handle this from Ruby.

Reply:
oops, i had controller's bashrc in mind, sorry

yes, also agree it's easier from ruby

ok (decoupling)

yeah, don't multiply the mechanisms, let's not add an export to the mix